packages/shared/src/components/cards/v1/SharedPostCardFooter.tsx

```diff
@@ -20,32 +20,26 @@ export const SharedPostCardFooter = ({
   return (
     <div
       className={classNames(
-        'mb-2 flex h-auto min-h-0 w-full flex-auto flex-col gap-3 rounded-12 border border-theme-divider-tertiary p-3 mobileL:flex-row',
+        'mb-2 flex h-auto min-h-0 w-full flex-auto flex-col gap-3 rounded-12 border border-theme-divider-tertiary p-3',
```

Comment:
What issue was this causing?

Reply:
This is not necessary, we don't render anything greater than mobileL.